ui/form.go

```diff
@@ -65,6 +65,9 @@ func (f Form) Run(r io.Reader, w io.Writer) (map[string]interface{}, error) {
 
 			color.Set(color.FgCyan)
 			s, err := field.GetContents(r)
+			if displayLabel == "Password" {
```

Comment:
This would be better off as a [type assertion](https://tour.golang.org/methods/15), ensuring that `field` is of type `PasswordField`.